build.sbt

```diff
@@ -53,7 +53,7 @@ lazy val serverDependencies = libraryDependencies ++= {
 lazy val commonSettings = Seq(
   organization := "com.advancedtelematic",
   scalaVersion := "2.12.4",
-  scalacOptions := Seq("-unchecked", "-deprecation", "-encoding", "utf8", "-Xexperimental", "-Ypartial-unification"),
+  scalacOptions := Seq("-unchecked", "-deprecation", "-encoding", "utf8", "-Xexperimental", "-Ypartial-unification", "-Ywarn-unused-import"),
```

Comment:
I don't think we should add this flag. The ide should do this for you and it's a pain in the ass in development seeing all those warnings, leads users to just ignore all warnings.

Reply:
ok, removed

Reply:
you can leave it on if it helps you. I can remove it when developing and add it back before commit.

Reply:
Can this be somehow set in the user config?